### PR TITLE
[v0.14 prep] Update syna to master and fix breaking changes and sync files

### PR DESCRIPTION
**Pull request by mpourismaiel**
**What this PR does / why we need it**:
- Updated syna to v0.14-alpha, waiting for v0.14 branch
- Sync syna directory
- Fix breaking changes

**Special notes for your reviewer**:
Please don't merge until we branch out the v0.14

### CODE REVIEW

File: content/_index/features/hosted.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
 title = "Hosted Service"
 weight = 20
-icon = "fas fa-cogs"
+
+[asset]
+  icon = "fas fa-cogs"
 +++
 
-Let us worry about the global and fault tolerant backend infrastructure while you control and manage your redirect data within your DNS records.  
+Let us worry about the global and fault tolerant backend infrastructure while you control and manage your redirect data within your DNS records.
```
Comment (stp-ip) [inline content/_index/features/hosted.md:9]: You definitely don't like intentional line breaks in markdown ;)

File: content/gopkg/features/hosted.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 title = "Globally hosted"
 weight = 20
-icon = "fas fa-cloud"
 
+[asset]
+  icon = "fas fa-cloud"
 +++
 
-Let us worry about the global and fault tolerant backend infrastructure while you control and manage your data within your DNS records.  
+Let us worry about the global and fault tolerant backend infrastructure while you control and manage your data within your DNS records.
```
Comment (stp-ip) [inline content/gopkg/features/hosted.md:9]: Intentional line break.

File: content/gopkg/single-bare.md [hunk 3]
```diff
@@ -7,14 +7,15 @@ background = "dark"
 align = "left"
 
 title = """
-Single repository import  
+Single repository import
```
Comment (stp-ip) [inline content/gopkg/single-bare.md:10]: Intentional line break.

File: content/hosted/host-root.md [hunk 4]
```diff
@@ -26,5 +27,5 @@ _redirect.txtdirect.org   86000 IN TXT     "v=txtv0;to=https://about.txtdirect.o
 ```
 
 ## Options
-`to=` sets the URL to redirect to  
+`to=` sets the URL to redirect to
```
Comment (stp-ip) [inline content/hosted/host-root.md:30]: Intentional line break.

File: content/hosted/host.md [hunk 5]
```diff
@@ -24,5 +25,5 @@ _redirect.www.txtdirect.org   86000 IN TXT     "v=txtv0;to=https://about.txtdire
 ```
 
 ## Options
-`to=` sets the URL to redirect to  
+`to=` sets the URL to redirect to
```
Comment (stp-ip) [inline content/hosted/host.md:28]: Intentional line break.

**Review by stp-ip — CHANGES_REQUESTED** [review]

File: content/_index/features/hosted.md (hunk 1 above)
Comment (mpourismaiel) [reply content/_index/features/hosted.md:9]: My vscode is configured to trim end of the line whitespace by default, I disable it per project but forgot for this project.

**Comment by stp-ip** [comment]
Line changes still need to be fixed here.

**Pull request merged**